src/components/PersonLink.tsx

```diff
@@ -0,0 +1,65 @@
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+
+  const { slug } = useParams();
+  const noParent = '-';
```

Comment:
```suggestion
  const NO_PARENT_LABEL = '-';
```
make it constant and move out of the component